features/shipped/FxA-71-AMO-migration-support/README.md

```diff
@@ -31,6 +31,27 @@ who enter the FxA login flow from AMO.
 
 These metrics are already available
 since we track them for other reliers.
+The specific query in datadog will be:
+
+* `login-attempts` = `fxa.content.screen.oauth.signin` + `fxa.content.screen.oauth.signup`
+* `successul-logins` = `fxa.content.screen.oauth.signin` + `fxa.content.screen.oauth.signup`
```

Comment:
`successul`->`successful`.

It also looks like both formulas are identical. successful logins should have a `.success` suffix.